File: Praktikum 2. Input, dan output/Soal 5/PRAK205-2310817310001-AvantioFierzaPatria.py
import math

# Membaca input A dan B dari pengguna
A, B = map(int, input("Masukkan nilai A dan B (pisahkan dengan spasi): ").split())

# Tinggi sama dengan salah satu sisi yang diketahui (A atau B)
tinggi = A
# Menghitung alas (C) menggunakan teorema Pythagoras
C = math.sqrt(B**2-A**2)
# Menghitung keliling
keliling = A + B + C
# Menghitung luas
luas = 0.5 * A * C

# Menampilkan hasil
print("Alas = {:.0f} cm".format(C))
print("Tinggi = {} cm".format(tinggi))
print("Keliling = {:.0f} cm".format(keliling))
print("Luas = {:.0f} cm^2".format(luas))

File: Praktikum 2. Input, dan output/Soal 5/PRAK205-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>
#include <math.h>

int main()
{
    // Mendeklarasikan variabel A, B, dan C
    double A, B, C;
    // Membaca input A dan B dari pengguna
    printf("Masukkan nilai A ? ");
    scanf("%lf", &A);
    printf("Masukkan nilai B ? ");
    scanf("%lf", &B);

    // Menghitung tinggi segitiga
    double tinggi = A;
    // Menghitung panjang sisi C menggunakan rumus Pythagoras
    C = sqrt(B * B - A * A);
    // Menghitung keliling segitiga
    double keliling = A + B + C;
    // Menghitung luas segitiga
    double luas = 0.5 * A * C;

    // Menampilkan hasil
    printf("Alas = %.0lf cm\n", C);
    printf("Tinggi = %.0lf cm\n", A);
    printf("Keliling = %.0lf cm\n", keliling);
    printf("Luas = %.0lf cm^2\n", luas);

    return 0;
}
